frontend/pages/hosts/details/HostDetailsPage/modals/RunScriptModal/RunScriptModal.tests.tsx

```
@@ -0,0 +1,187 @@
+      render(
+        <RunScriptModal
+          {...baseProps}
+          currentUser={adminUser}
+          hostScriptResponse={{
+            scripts: [createMockHostScript({ name: "cleanup.sh" })],
+            meta: { has_next_results: false },
+          }}
+        />
+      );
+
+      expect(screen.queryByText("No scripts available")).not.toBeInTheDocument();
```

ESLint

[error] Replace `screen.queryByText("No·scripts·available")` with `⏎········screen.queryByText("No·scripts·available")⏎······`

(prettier/prettier)